### PR TITLE
Add static file download icon and box

### LINKED ISSUE #1065: [website] Add download icon next to file names
**What would you like to be added**:

We have a shortcode that writes the contents of a static file into the website, along with a link for downloading it.
There should be:
- an icon before the file name.
- a box around the name

https://kubernetes.io/docs/concepts/workloads/controllers/job/#running-an-example-job

Perhaps we can adapt the shortcode that kubernetes uses https://github.com/kubernetes/website/blob/3c9c3afd2abd086f79b7d220884f39747d06496c/layouts/shortcodes/code.html

**Why is this needed**:

An icon would make it clear that it's a link for downloading the file

**Completion requirements**:

This enhancement requires the following artifacts:

- [ ] Design doc
- [ ] API change
- [ ] Docs update

The artifacts should be linked in subsequent comments.

### PR DESCRIPTION
**Pull request by FZhg**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #1065 

#### Special notes for your reviewer:
N/A

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by FZhg** [comment]
> Can you put the button on the right side and maybe reduce the padding on the bottom?
> 
> The kubernetes website looks like this:
> 
> image
> 
> It doesn't have to look the same, but my suggestions above should give a more clear indication that the button refers to the contents.

Of course I will work on this.

File: site/layouts/shortcodes/include.html [hunk 1]
```diff
@@ -1,6 +1,16 @@
-{{ $file := .Get 0 }}
-[{{$file}}]({{printf "/examples/%s" $file}})
-{{ $path := (printf "static/examples/%s" $file)}}
-```{{ .Get 1 }}
+{{ $file := .Get "filename" }}
+{{ $codelang := .Get "codelang"}}
+{{ $path := (printf "static/examples/%s/%s" $codelang $file )}}
+{{$ghlink := printf "/examples/%s/%s" $codelang $file}}
```
Comment (alculquicondor) [inline site/layouts/shortcodes/include.html:4]: codelang will not necessarily be part of the filename.

See #1078, for example

Comment (FZhg) [reply site/layouts/shortcodes/include.html:4]: Thanks! I will change this part. you are right!

**Comment by tenzen-y** [comment]
Replaced with #1118.

**Pull request closed without merging**